bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/Wavelength.scala

```diff
@@ -103,6 +70,10 @@ object Wavelength {
   implicit val WavelengthOrdering: scala.Ordering[Wavelength] =
     scala.Ordering.by(_.toNanometers)
 
-
+  /** @group Implicit Conversions */
+  implicit def fromLength(l: Length): Wavelength = {
```

Comment:
Ok, I'll look into this. However, as long as `Wavelength` is a value class there's not much we can do to keep people from calling `Wavelength(crazyLength)` directly (?). But I guess we can live with that.